test/colony-task-work-rating.js

```diff
@@ -69,11 +74,12 @@ contract("Colony Task Work Rating", accounts => {
       const taskId = await setupAssignedTask({ colonyNetwork, colony, dueDate });
 
       await colony.submitTaskDeliverableAndRating(taskId, DELIVERABLE_HASH, RATING_1_SECRET, { from: WORKER });
+      const currentTime2 = await currentBlockTime();
       const ratings = await colony.getTaskWorkRatings(taskId);
-      assert.equal(ratings[0].toNumber(), 1);
-      assert.closeTo(ratings[1].toNumber(), currentTime, 2);
+      expect(ratings.nSecrets).to.eq.BN(1);
+      expect(ratings.lastSubmittedAt).to.eq.BN(currentTime2);
```

Comment:
Can we keep the `closeTo` here, which `chai` supports. This can be brittle with blocktime variations.